cmd/README.md

```diff
@@ -0,0 +1,521 @@
+data conversion can be found here:
+- [PostgreSQL data conversion](sources/postgres/README.md#data-conversion)
+- [MySQL data conversion](sources/mysql/README.md#data-conversion)
+- [DynamoDB data conversion](sources/dynamodb/README.md#data-conversion)
+
+## Troubleshooting Guide
+
+HarbourBridge is written using the Go module system, and so it must be
+installed in [module-aware mode](https://golang.org/cmd/go/#hdr-Module_support).
+This can be achived by setting the environment variable `GO111MODULE` to `on`
+before calling "go get" e.g.
+
+```sh
+GO111MODULE=on go get github.com/cloudspannerecosystem/harbourbridge
+```
+
+As an alternative to "go get", you can make a copy of the HarbourBridge
+codebase from the github repository and use "go run":
+
+```sh
+git clone https://github.com/cloudspannerecosystem/harbourbridge
+cd harbourbridge
+pg_dump mydb > mydb.pg_dump
+go run github.com/cloudspannerecosystem/harbourbridge schema -source=postgresql < mydb.pg_dump
+```
+
+This workflow also allows you to modify or customize the HarbourBridge codebase.
+
+The following steps can help diagnose common issues encountered while running
+HarbourBridge.
+
+### 1. Verify driver configuration
+
+First, check that the driver is correctly configured to connect to your
+database. Driver configuration varies depending on the driver used.
+
+#### 1.1 pg_dump
+
+If you are using pg_dump (-driver=pg_dump), check that pg_dump is
+correctly configured to connect to your PostgreSQL
+database. Note that pg_dump uses the same options as psql to connect to your
+database. See the [psql](https://www.postgresql.org/docs/9.3/app-psql.html) and
+[pg_dump](https://www.postgresql.org/docs/9.3/app-pgdump.html) documentation.
+
+Access to a PostgreSQL database is typically configured using the
+_PGHOST_, _PGPORT_, _PGUSER_, _PGDATABASE_ environment variables,
+which are standard across PostgreSQL utilities.
+
+It is also possible to configure access via pg_dump's command-line options
+`--host`, `--port`, and `--username`.
+
+#### 1.2 Direct access to PostgreSQL
+
+In this case, HarbourBridge connects directly to the PostgreSQL
+database to retrieve table schema and data. Set the `-driver=postgres`
+and provide the environment variables `PGHOST`, `PGPORT`, `PGUSER`, 
+`PGDATABASE`. Password can be specified either in the `PGPASSWORD`
+environment variable or provided at the password prompt.
+
+#### 1.3 mysqldump
+
+If you are using mysqldump (-driver=mysqldump), check that mysqldump is
+correctly configured to connect to your MySQL
+database via the command-line options `--host`, `--port`, and `--user`.
+Note that mysqldump uses the same options as mysql to connect to your
+database. See the [mysql](https://dev.mysql.com/doc/refman/8.0/en/mysql-commands.html)
+and [mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) documentation.
+
+#### 1.4 Direct access to MySQL
+
+In this case, HarbourBridge connects directly to the MySQL
```

Comment:
same as postgres. Move these flags into the source profile

Reply:
I plan to clean up the READMEs for environment variables as part of resolving the TODO in NewSourceProfileConnectionMySQL in source_profile.go.